ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -17,19 +17,29 @@ class SampleHandler: RPBroadcastSampleHandler {
     var videoSource: ReplayKitVideoSource?
     var screenTrack: LocalVideoTrack?
     var disconnectSemaphore: DispatchSemaphore?
-    let audioDevice = ExampleReplayKitAudioCapturer(sampleType: SampleHandler.kAudioSampleType)
+    let audioDevice = ExampleReplayKitAudioCapturer(sampleType: SampleHandler.kAudioSampleType ?? RPSampleBufferType.audioMic)
 
     var accessToken: String = "TWILIO_ACCESS_TOKEN"
     let tokenUrl = "http://127.0.0.1:5000/"
 
     var statsTimer: Timer?
     static let kBroadcastSetupInfoRoomNameKey = "roomName"
 
-    // Which kind of audio samples we will capture. The example does not mix multiple types of samples together.
-    static let kAudioSampleType = RPSampleBufferType.audioMic
+    // Which kind of audio samples the handler will capture. The example does not support mixing.
+    // Pass `nil` to disable audio publishing and save ~512 KB of memory.
+    static let kAudioSampleType: RPSampleBufferType? = RPSampleBufferType.audioMic
+
+    // If the content should be treated as screencast (detail) or regular video (motion). Used to configure ReplayKitVideoSource.
+    static let isScreencast = true
 
     // The video codec to use for the broadcast. The encoding parameters and format request are built dynamically based upon the codec.
-    static let kVideoCodec = H264Codec()!
+    static let kVideoCodec: VideoCodec = UIDevice.current.userInterfaceIdiom == .pad ?
+        H264Codec() : Vp8Codec(simulcast: false)
+
+    // Frame skip, deliver a frame once every `kScreencastFrameSkip` frames.
+    static let kScreencastFrameSkip = Int64(2)
```

Comment:
I personally like setting hard higher and lower bound for frame rates. But if you think this is better, I am fine with it.

Reply:
I only chose to use a skip since we don't have an easy way to get the frame rate at this time (it would require processing the frame, which I don't want to do).